addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -285,10 +285,15 @@
                         <div t-if="job.address_id" class="d-flex flex-column align-items-baseline">
                             <span class="text-muted small">Location</span>
                             <h6 t-field="job.address_id" t-options='{
+                                "widget": "contact",
                                 "fields": ["city"],
                                 "no_tag_br": True
                             }'/>
                         </div>
+                        <div t-if="not job.address_id" class="d-flex flex-column align-items-baseline">
+                            <span class="text-muted small">Location</span>
+                            <h6>Remote</h6>
```

Comment:
```py
#: model_terms:ir.ui.view,arch_db:website_hr_recruitment.apply
```
should probably be added to "Remote" term in addons/website_hr_recruitment/i18n/website_hr_recruitment.pot, otherwise I don't think this will be translated